tools/cppcheck.yml

```diff
@@ -6,7 +6,19 @@ services:
     image: ghcr.io/illarion-ev/illarion-server-tools-cppcheck
     volumes:
       - type: bind
-        source: ../src
+        source: ..
         target: /src
-    command: sh -c "cppcheck --std=c++17 --enable=warning,style,performance,portability --inconclusive -j $$(nproc) --error-exitcode=1 -q --suppress=functionStatic:db/InsertQuery.hpp -I. ."
+    command: sh -c "cppcheck --std=c++17 --enable=warning,style,performance,portability --inconclusive -j $$(nproc) --error-exitcode=1 -q --suppress=functionStatic:src/db/InsertQuery.hpp -Isrc/ src/"
+    network_mode: none
+
+  cppcheck-xml:
```

Comment:
Now you are running cppcheck two times. How long does cppcheck take to perform it's tests?

If it takes a long time, it may be better to run it once and create the console output from the XML.

Reply:
It only runs once, because the scripts never run `docker-compose up`, only `docker-compose run <service>`.